resources/hooks/local/pre-push

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+GIT_CURRENT_BRANCH=$(git name-rev --name-only HEAD)
+
+# Fetch the GIT diff and format it as command input:
+DIFF=git diff --name-only origin/"$GIT_CURRENT_BRANCH"..HEAD --oneline
```

Comment:
good point, the problem that we need a point for to the difference.
probably it's better to find another command